ghost/core/core/server/data/migrations/versions/5.83/2024-05-16-17-57-13-add-activitypub-table.js

```diff
@@ -0,0 +1,12 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('prototype_activitypub', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    url: {type: 'string', maxlength: 255, nullable: false, unique: true},
```

Comment:
2k is throwing an error. I assume it's because I made the column `unique`, which I thought it should be. But maybe I'm wrong. I changed it to 1k now

Reply:
Actually, it was the `unique`, so I put it back to 2k and dropped the `unique` proeprty.

Reply:
This is insane. Passed locally just fine but not here (even running `yarn test:ci:e2e`). Reducing back to 1k 😔

Reply:
ffs.. ok. 3027 bytes is allowed, for UTF-8 charset, that's 768 chars, so that's what I used now

Reply:
Okay I think being able to fit the URL in is more important than the unique constraint - because otherwise we have to drop data if the URL is too long!

We can leave as-is for now while we're prototyping, but should try and find a solution for this - perhaps storing the hash of the URL in a unique column